$OpenBSD: patch-etc_netbeans_conf,v 1.3 2010/11/12 12:26:16 jasper Exp $

Turn autoupdate off by default. The relevant options are documented here:
	http://wiki.netbeans.org/FaqPluginManagerCustomization

--- etc/netbeans.conf.orig	Thu Jul 29 02:06:10 2010
+++ etc/netbeans.conf	Tue Nov  9 21:48:58 2010
@@ -3,7 +3,7 @@ netbeans_default_userdir="${HOME}/.netbeans/6.9"
 
 # Options used by NetBeans launcher by default, can be overridden by explicit
 # command line switches:
-netbeans_default_options="-J-client -J-Xss2m -J-Xms32m -J-XX:PermSize=32m -J-XX:MaxPermSize=200m -J-Dapple.laf.useScreenMenuBar=true -J-Dapple.awt.graphics.UseQuartz=true -J-Dsun.java2d.noddraw=true"
+netbeans_default_options="-J-client -J-Xss2m -J-Xms32m -J-XX:PermSize=32m -J-XX:MaxPermSize=200m -J-Dapple.laf.useScreenMenuBar=true -J-Dapple.awt.graphics.UseQuartz=true -J-Dsun.java2d.noddraw=true -J-Dplugin.manager.check.updates=false -J-Dplugin.manager.check.interval=NEVER"
 # Note that a default -Xmx is selected for you automatically.
 # You can find this value in var/log/messages.log file in your userdir.
 # The automatically selected value can be overridden by specifying -J-Xmx here
